README.md

```diff
@@ -108,8 +108,14 @@ export DEV_MODE=1
 # Migrate DB
 ./scripts/dev dbmigrate
 
+
+# create a realm and note the return number
+# successfully created realm default (1)
+go run ./cmd/add-realm --name "default"
```

Comment:
so dbmigrate should create an initial realm called "Default"  with ID of 1

I think this command can be dropped in favor of a comment that says as much